shared-libs/message-utils/package-lock.json

```diff
@@ -31,7 +31,17 @@
       }
     },
     "@medic/phone-number": {
-      "version": "file:../phone-number"
+      "version": "file:../phone-number",
+      "requires": {
+        "google-libphonenumber": "^3.2.27"
+      },
+      "dependencies": {
+        "google-libphonenumber": {
+          "version": "3.2.30",
+          "resolved": "https://registry.npmjs.org/google-libphonenumber/-/google-libphonenumber-3.2.30.tgz",
+          "integrity": "sha512-Kx2/AqmY0P6863vOhkiCFqfAxfY3jagMe916ByU38JRKiRCqSHGJW1qTOZNV4+ag8Xda69dk6w8VwEeswVy44w=="
+        }
+      }
```

Comment:
You need to revert this section. (out of all the package-lock files). this is causing your `npm ci` to fail